README.md

```diff
@@ -1 +1,40 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello-world
+
+> Description the project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**Go to the a repository on [github](https://github.com/anasbassi/Hello-world.git).**
+**Fork the repository to your own github."" 
+**Copy the https link and clone it on your local device.**
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Anas Abubakar Bassi**
+
+- GitHub: [@anasbassi](https://github.com/anasbassi)
+- Twitter: [@anasabu92568449](https://twitter.com/anasabu92568449)
+- LinkedIn: [Anas Abubakar](https://linkedin.com/in/anas-abubakar-7b352722b)
+
+## Acknowledgments
+
+- Microverse
+- Piesie
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] I think it would be better if you add the `MIT.md` license file in your root directory, to make this link work. From now the link is broken but you can fix it by adding :point_up: the file I pointed out above in your project. Copy it from this [template](https://github.com/microverseinc/readme-template) and you can edit the file to make it more professional.